content/en/_index.md

```diff
@@ -165,6 +200,42 @@ more approachable to beginners:
      }
    }
  {{< /tab >}}
+  {{< tab header="Node.js" >}}
+    // Imports the Google Cloud Data Loss Prevention library
+    const DLP = require('@google-cloud/dlp');
+
+    // Instantiates a reusable client object
+    const dlp = new DLP.DlpServiceClient();
+
+    // Lists the types of sensitive information the DLP API supports.
+    async function listInfoTypes() {
+      // ---------------- ARRANGE ----------------
```

Comment:
I kinda like this - @grayside what do you think about adding these headers to the Java sample?

I'm a little worried folks might think the need to add this comments verbatim though. Maybe clarify in the comment or description for this bullet point?

Reply:
> I'm a little worried folks might think the need to add this comments verbatim though.

Added a paragraph about this before the samples themselves.

Reply:
We chatted about this in the working group yesterday and the consensus was that we don't want these comments in the snippet for now.

Reply:
OK, I removed them.